rest_framework/schemas/openapi.py

```diff
@@ -146,15 +148,15 @@ def get_operation(self, path, method):
         operation['description'] = self.get_description(path, method)
 
         parameters = []
-        parameters += self._get_path_parameters(path, method)
-        parameters += self._get_pagination_parameters(path, method)
-        parameters += self._get_filter_parameters(path, method)
+        parameters += self.get_path_parameters(path, method)
+        parameters += self.get_pagination_parameters(path, method)
+        parameters += self.get_filter_parameters(path, method)
         operation['parameters'] = parameters
 
-        request_body = self._get_request_body(path, method)
+        request_body = self.get_request_body(path, method)
         if request_body:
             operation['requestBody'] = request_body
-        operation['responses'] = self._get_responses(path, method)
+        operation['responses'] = self.get_responses(path, method)
```

Comment:
@carltongibson 

Assume someone has overridden the `_get_responses` method, will this change break the functionality?

Reply:
No. Underscored variants are all moved to the bottom and deprecated. They’ll be removed in 3.14

Reply:
Oh, I see what you mean. Yes it probably will. They’ll need to adjust. 

I’ll make sure to call it out in the release announcement.

Reply:
I think you got my point. Still to make things more clear let me tell you one use case.

Consider below example for overridden `_get_responses` method.

```python

class MyAutoSchema(AutoSchema):
    def _get_responses(...)
        ...

class MyView(ViewSet):
    schema = MyAutoSchema()
```

In DRF 3.11 this will use `_get_responses` of `MyAutoSchema`.
For the master branch, because of the above changes, It will use `get_responses` of `AutoSchema`. 
This makes the current master branch backward incompatible.

Reply:
Sure... Not much we can do about this. That's the reason the methods were underscored to begin. From 3.12, they'll be public and subject to the deprecation policy.